client/src/helpers/qnAHelper.js

```diff
@@ -295,25 +295,20 @@ module.exports = {
 
   },
 
-  answerScrollClass(count){
-    let newClass;
-
-    if (count % 2 !== 0) {
-      newClass = 'list scroll container'
-    }
-
-    return newClass
-  },
-
   showQuestionsClass(clickCount, index) {
-
-    let newClass;
-    if (clickCount % 2 !== 0) {
-      if (index <= clickCount) {
-        newClass = 'questionText'
-      }
+    let show= false;
+    if (index <= clickCount) {
```

Comment:
Could be a good place for a ternary. `index <= clickCount ? show = true : show = false`